source/StepBro.Core/Parser/StepBroListener.Procedure.cs

```diff
@@ -328,7 +329,21 @@ public override void ExitCallStatement([NotNull] SBP.CallStatementContext contex
             var propertyBlock = m_lastElementPropertyBlock;
             left = this.ResolveIfIdentifier(left, true);        // Now done in EnterMethodArguments() above.
 
-            this.HandleParensExpression(context, true, left, argumentStack, null, propertyBlock);
+            try
+            {
+                this.HandleParensExpression(context, true, left, argumentStack, null, propertyBlock);
+            }
+            catch (NotImplementedException e)
```

Comment:
This could easily catch any kind of exception, and add the info to a parsing error instead. 
If the try-catch is moved into the function, it works for all the locations where it is called from.